exporters/mongodb3/mongodb3-config.yml.sample

```diff
@@ -39,7 +39,7 @@ integrations:
         exporter_port: 9126
 
         # How long until a scrape request times-out (defaults to 5s)
-        # scrape_timeout: 5s
+        scrape_timeout: 30s
```

Comment:
the comment above needs to be updated as well 😄

Reply:
Moreover, this is not a commented out anymore, is it intended?

Reply:
Yes the idea is to increase the [default](https://github.com/newrelic/nri-prometheus/blob/3027d9740dfad31ce8f87073e7331c1e20850c6a/cmd/nri-prometheus/config.go#L95) from nri-prometheus (5s) for this integration which usually exceeds that.
Since in this case nri-promethus is just scraping this endpoint in a default interval of 30s why not to extend the scrape_timout to that value and save users from getting this error. 
Not sure about modifying the comment since the default is still 5s

Reply:
Oh I understand 🤔 
I would specify then that " the default of nri-prometheus is 5s, this option overwrite it making it 30".

Reply:
I think It might be confusing since this config file "abstacts" nri-prometheus from the equation. What is important IMO is that if this config has not value the default will be 5s